Update some video versions messages

CrystalP opened the pull request:
## Description
<!--- Provide a general summary of your change in the Pull Request title above -->
<!--- Describe your change in detail here. -->
Follow up of the issue #24294, some changes were made in another PR already, this one changes the rest of the messages that used a variable for the word "movie" or "extras".

The only code changes are adaptations of the variables passed to `StringUtils::Format()`. No change in the logic, this is purely about the messages.

Also used the opportunity to touch up a few of the messages in English.

## Motivation and context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
Feedback in the #translations-external Slack channel that some of the messages were difficult to translate because of a variable that stood for 'movie' or 'extra'. Going forward the variable could have taken other unpredictable values, with impact on translation.

Minor text adjustments in English.

## How has this been tested?
<!--- Please describe in detail how you tested your change. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
Ran Kodi with the messages.
Some messages are for error conditions that rarely happen or cannot happen anymore due to changes of the dialog.

## Types of change
<!--- What type of change does your code introduce? Put an `x` with no space in all the boxes that apply like this: [X] -->
- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [X] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **Student submission** (PR was done for educational purposes and will be treated as such)
- [ ] **None of the above** (please explain below)

## Checklist:
<!--- Go over all the following points, and put an `X` with no space in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed
``

ksooo reviewed (approved):
Much cleaner now.

ksooo commented on addons/resource.language.en_gb/resources/strings.po:
```diff
@@ -23751,11 +23751,7 @@ msgctxt "#40001"
-msgctxt "#40002"
-msgid "Convert into an additional version of {0:s}"
-msgstr ""
+#obsolete msgid 40002, replaced by msgid 40026
```
We usually do not document this kind of change, because we have GitHub. ;-)
And I think translators cannot see these comments in Weblate anyway.

ksooo reviewed (commented):
I would remove the "obsolete", replaced by comments AND I would just reuse the exiting ids. There is no benefit for translators. They must (re-)translate completely in either case.

CrystalP committed:
[Messages] revision of some of the video versions messages

The word "movie" or "extra" as a parameter in messages complicated the translation.

Rewrote messages without the parameter, used the opportunity to rewrite a few of the english messages.
No application logic changes.

CrystalP commented:
Alright, rewritten with the existing msgctxt.

ksooo reviewed (approved):
LGTM now.

ksooo commented:
@CrystalP 

Hi, sorry, right after you merged your messages PR, I spotted something that should be changed. You introduced several new code of the form `StringUtils::Format(g_localizeStrings.Get(msgid))`.

The `StringUtils::Format` can and should be removed here, as the translatable string does not contain any placeholders anymore.

At some places you now even can use `CVariant{msgid}` instead of `g_localizeStrings.Get(msgid)`, for example with `CGUIDialogOK::ShowAndGetInput`:

`CGUIDialogOK::ShowAndGetInput(CVariant(40018), StringUtils::Format(g_localizeStrings.Get(40019)));`
=>
`CGUIDialogOK::ShowAndGetInput(CVariant{40018}, CVariant{40019});`

ksooo commented on xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp:
```diff
@@ -134,12 +134,8 @@ void CGUIDialogVideoManagerVersions::Remove()
-                            CMediaTypes::GetLocalization(mediaType),
-                            m_videoAsset->GetVideoInfoTag()->GetTitle()));
+    CGUIDialogOK::ShowAndGetInput(CVariant(40018),
+                                  StringUtils::Format(g_localizeStrings.Get(40019)));
```
=> `CVariant{40019});`

ksooo commented on xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp:
```diff
@@ -247,10 +240,8 @@ void CGUIDialogVideoManagerVersions::AddVideoVersion()
-              StringUtils::Format(g_localizeStrings.Get(40019), typeVideoVersion,
-                                  CMediaTypes::GetLocalization(mediaType), videoTitle));
+          CGUIDialogOK::ShowAndGetInput(g_localizeStrings.Get(40014),
+                                        StringUtils::Format(g_localizeStrings.Get(40019)));
```
=> `CVariant{40019});`

ksooo commented on xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp:
```diff
@@ -247,10 +240,8 @@ void CGUIDialogVideoManagerVersions::AddVideoVersion()
-              g_localizeStrings.Get(40014),
-              StringUtils::Format(g_localizeStrings.Get(40019), typeVideoVersion,
-                                  CMediaTypes::GetLocalization(mediaType), videoTitle));
+          CGUIDialogOK::ShowAndGetInput(g_localizeStrings.Get(40014),
```
=> ...Input(`CVariant{40014},`

ksooo commented on xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp:
```diff
@@ -399,9 +390,8 @@ bool CGUIDialogVideoManagerVersions::ConvertVideoVersion(const std::shared_ptr<C
-        CVariant{40005},
-        StringUtils::Format(g_localizeStrings.Get(40006), CMediaTypes::GetLocalization(mediaType)));
+    CGUIDialogOK::ShowAndGetInput(CVariant{40005},
+                                  StringUtils::Format(g_localizeStrings.Get(40006)));
```
=> `CVariant{40006});`

ksooo commented on xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp:
```diff
@@ -508,11 +497,10 @@ bool CGUIDialogVideoManagerVersions::ProcessVideoVersion(VideoDbContentType item
-                              CMediaTypes::GetLocalization(mediaType)),
-          StringUtils::Format(g_localizeStrings.Get(40009), CMediaTypes::GetLocalization(mediaType),
-                              item.GetVideoInfoTag()->GetTitle(), path)))
+  if (!CGUIDialogYesNo::ShowAndGetInput(StringUtils::Format(g_localizeStrings.Get(40008)),
```
=> `...Input(CVariant{40008},`

ksooo commented on xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp:
```diff
@@ -544,8 +532,7 @@ bool CGUIDialogVideoManagerVersions::ProcessVideoVersion(VideoDbContentType item
   dialog->SetItems(list);
-  dialog->SetHeading(
-      StringUtils::Format(g_localizeStrings.Get(40002), CMediaTypes::GetLocalization(mediaType)));
+  dialog->SetHeading(StringUtils::Format(g_localizeStrings.Get(40002)));
```
=> `...Heading(CVariant{40002});`
